### DISCUSSION TITLE
I'm facing an issue on getting Ethereum Sepolia Faucet

### DISCUSSION THREAD

**Discussion by Usama-Akbar** [Cyfrin/foundry-full-course-cu#3074] [Q&A] [answered]
<img width="1394" alt="Screenshot 2024-12-19 at 3 31 14 PM" src="https://github.com/user-attachments/assets/2559883f-aaec-4fc4-abd2-c925cd76b5a3" />
I've made the wallet and switched to sepolio and while getting sepolio eth its giving the above error

**Comment by EngrPips** [comment, marked as answer]
Yeah, it has been tough to get those lately. Please try this https://cloud.google.com/application/web3/faucet . You might have to wait a little bit to retry if it fails on the first try.

**Reply by Usama-Akbar** [reply]
sure I'll check that, Thanks :)

**Reply by Usama-Akbar** [reply]
it worked with google Appreciate that.

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.

**Reply by veyrix-Tr** [reply]
![image](https://github.com/user-attachments/assets/6a307e21-a3f1-42f8-9e05-dcca90b1a92e)
it is asking for 0.001 Ethereum. How will get it if I am unable to find ether from any faucet

**Reply by EngrPips** [reply]
You can retry at a later time, and it should work.

**Reply by veyrix-Tr** [reply]
> You can retry at a later time, and it should work.

I tried it  yesterday too and now too but same result

**Reply by EngrPips** [reply]
Tough, I am unsure what could be wrong. maybe try this https://sepolia-faucet.pk910.de/

**Comment by khandelwalmoksh** [comment]
@Usama-Akbar @chiraggoyal4639  u can try https://chainlist.org/?testnets=true

**Reply by EngrPips** [reply]
great heads-up.
